packages/ui/primitives/document-flow/add-subject.types.ts

```diff
@@ -2,13 +2,20 @@ import { z } from 'zod';
 
 import { DEFAULT_DOCUMENT_DATE_FORMAT } from '@documenso/lib/constants/date-formats';
 import { DEFAULT_DOCUMENT_TIME_ZONE } from '@documenso/lib/constants/time-zones';
+import { URL_REGEX } from '@documenso/lib/constants/url-regex';
 
 export const ZAddSubjectFormSchema = z.object({
   meta: z.object({
     subject: z.string(),
     message: z.string(),
     timezone: z.string().optional().default(DEFAULT_DOCUMENT_TIME_ZONE),
     dateFormat: z.string().optional().default(DEFAULT_DOCUMENT_DATE_FORMAT),
+    redirectUrl: z
```

Comment:
Same here

Reply:
The `z.string().url()` seems to have a problem, so I thought of doing it this way. There's an open PR [here](https://github.com/colinhacks/zod/pull/3049) to fix the URL validation in zod

More Info on the issue: https://github.com/colinhacks/zod/issues/2236